partition-pruning.md

```diff
@@ -0,0 +1,259 @@
+# Partition pruning
```

Comment:
Please include the header metadata. For h1 titles, I believe we also capitalize the whole title:

```suggestion
# Partition Pruning
```